example/gardener-local/kind/cluster/values.yaml

```
@@ -1,4 +1,4 @@
-image: kindest/node:v1.31.4
+image: kindest/node:v1.31.2
```

Should we follow the suggestion from the [kind release notes](https://github.com/kubernetes-sigs/kind/releases/tag/v0.26.0) at least now? 😄 

> NOTE: You must use the @sha256 digest to guarantee an image built for this release, until such a time as we switch to a different tagging scheme. Even then we will highly encourage digest pinning for security and reproducibility reasons.

```suggestion
image: kindest/node:v1.31.2@sha256:18fbefc20a7113353c7b75b5c869d7145a6abd6269154825872dc59c1329912e
```